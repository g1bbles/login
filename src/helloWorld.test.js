import React from 'react';
import ReactDOM from 'react-dom';
import HelloWorld from './HelloWorld';
import { shallow, mount, configure } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });

it('renders and matches our snapshot', () => {
    const component = renderer.create(
        <HelloWorld  />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
