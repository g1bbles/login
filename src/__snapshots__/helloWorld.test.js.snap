exports[`test renders and matches our snapshot 1`] = `
<div
  className="HelloWorld">
  Hello World!
</div>
`;
